src/components/search/curated/next/index.tsx

```diff
@@ -15,6 +14,20 @@ const SearchNext = ({topic}: any) => {
   const advancedConcepts = get(topic, 'advancedConcepts')
   const portfolioProjects = get(topic, 'portfolioProjects')
 
+  if (!portfolioProjects)
+    return (
+      <div>
+        <SearchCuratedEssential
+          topic={{
+            label: 'Next.js',
+            name: 'next',
+            description: `Next.js gives you the best developer experience with all the features you need for production: hybrid static & server rendering, TypeScript support, smart bundling, route pre-fetching, and more. No config needed.`,
+          }}
+          CTAComponent={EcommerceCTA}
+        />
+      </div>
+    )
+
```

Comment:
Added this in for when the sanity data won't load, this will load the hardcoded data instead. When the sanity loading issue is fixed, I'll be able to remove this.